atc/metric/gauge.go

```diff
@@ -22,6 +22,19 @@ func (c *Gauge) Inc() {
 	}
 }
 
+func (c *Gauge) Set(val int64) {
```

Comment:
@vito I'm not really sure if I did this right 😬 how does this sound?

Reply:
I don't think this one *had* to change, since the previous metric was already emitted periodically by virtue of there only being one checker processor which itself runs periodically. So that's why this functionality didn't exist before.

I'm OK with either way though since from a pure code standpoint it's kind of nice to just have simple var definitions whose values get updated; it feels more Prometheus-style (...even though the way they currently end up getting to Prometheus is very roundabout). 👍